src/lib/containers/widgets/FileHandleLink.tsx

```diff
@@ -45,7 +45,12 @@ export const FileHandleLink = (props: FileHandleLinkProps) => {
             fileAssociateType,
             fileAssociateId,
             redirect,
-          )
+          ).then(url => {
+            window.open(url, '_blank')
```

Comment:
SynapseClient.getActualFileHandleByIdURL() used to have this logic, but I want to make the same call with different success handling, so I changed it to return a Promise instead.